app/js/states/inventory/inventory.controller.js

```diff
@@ -70,8 +70,14 @@ function InventoryCtrl(
     $scope.loading = InventoryService.loading;
     $scope.reloading = false;
 
-    $scope.predicate = $location.search().sort_field || 'toString';
-    $scope.reverse = $location.search().sort_dir === 'DESC';
+    InventoryService.setSortField($location.search().sort_field || 'toString');
+    InventoryService
+        .setSortDirection($location.search().sort_dir);
+
+    $scope.sorter = new Utils.Sorter({
+        predicate: InventoryService.getSortField(),
+        reverse: booleanSortDirection()
+    }, order);
```

Comment:
We should get rid of references to InventoryService sorting fields and use a default predicate stored in inventory.controller.js if there is no url param